src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,54 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        WorkWithFile workWithFile = new WorkWithFile();
+        String report = workWithFile.createReportFromCsvFile(fromFileName);
+        workWithFile.writeToFile(toFileName, report);
+    }
+
+    private String createReportFromCsvFile(String fileName) {
```

Comment:
This method from its name should create report, but instead it, reads from file, calculates buy and supply and creates report. It will be better to separate this responsibilities to 3 method first will be calculating buy and supply, second  creating report, and third will read from file